### ISSUE TITLE
resource names are invalid / deprecated

### ISSUE THREAD

**Issue by mutt13y** [cloudflare/cf-terraforming#45]
Resource names are recommended to be valid identifiers therefore they should not start with a number. Where you are using UUID type names as the resource identifier they can start with a digit.
Related to 
https://github.com/cloudflare/cf-terraforming/issues/33
https://github.com/hashicorp/terraform/issues/19919

I recommend that you prefix the resource names with an underscore.

**Comment by mutt13y** [comment]
I can confirm that TF0.12-beta-1 rejects all these names that start with a digit.
with 
```
Error: Invalid resource name
A name must start with a letter and may contain only letters, digits, underscores, and dashes.
```

**Comment by patryk** [comment]
Should be fixed by 098b9d888c74acc907e46c72fdff4ee9aa6f4cf6.
